src/sonic-yang-models/yang-models/sonic-flex_counter.yang

```diff
@@ -211,8 +211,53 @@ module sonic-flex_counter {
                 }
             }
 
+            container FLOW_CNT_ROUTE {
+                /* ROUTE_FLEX_COUNTER_GROUP */
+                leaf FLEX_COUNTER_STATUS {
+                    type flex_status;
+                }
+                leaf FLEX_COUNTER_DELAY_STATUS {
+                    type flex_delay_status;
+                }
+                leaf POLL_INTERVAL {
+                    type poll_interval;
+                }
+            }
+
         }
         /* end of container FLEX_COUNTER_TABLE */
+
+        container FLOW_COUNTER_ROUTE_PATTERN {
+            description "Flow counter route pattern of config_db.json";
+
+            list PATTERN_LIST {
+
+                key "vrf_name ip_prefix";
+
+                leaf vrf_name {
+                    /*
+                    We don't use vrf_name reference here because:
+                    1. User is allowed to configure a VRF that does not exist yet here, orchagent is designed to resolve the VRF name once the VRF is created.
+                    2. The field vrf_name accept both VRF name and VNET name.
+                    */
+                    type string {
+                        length 0..16;
+                    }
+                }
+
+                leaf ip_prefix {
+                    type stypes:sonic-ip-prefix;
```

Comment:
sonic-ip-prefix is used when host bits are present with '/prefixlen'.
i.e. 11.22.33.44/24 ,
inet one is used for 11.22.33.0/24
Hope the type aligned with your intention here.